packages/app/src/actions/court.js

```diff
@@ -78,11 +80,11 @@ const CourtActions = {
     }
   },
 
-  heartbeat(transitions) {
+  heartbeat() {
```

Comment:
Wonder if it would still be useful to make this configurable; perhaps the CLI would have a flag to say how many transitions it would allow.

Reply:
Note that it is (see https://github.com/aragonone/court-backend/pull/62/files#diff-5545618aa375119dd0b2348de4adc9a7R87), I think you're mixing the frontend action which is not configurable for the moment with the backend helper

Reply:
Ah right, gotcha. Nevermind then!